src/DssSpell.sol

```diff
@@ -23,29 +23,19 @@ import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./DssSpellCollateral.sol";
 
-interface GemLike {
-    function allowance(address, address) external view returns (uint256);
-    function approve(address, uint256) external returns (bool);
-    function transfer(address, uint256) external returns (bool);
-}
-
-interface RwaUrnLike {
-    function lock(uint256) external;
-}
-
-interface VestLike {
-    function restrict(uint256) external;
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-}
-
 contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/ef7c8c881c961e9b4b3cc9644619986a75ef83d7/governance/votes/Executive%20vote%20-%20October%205%2C%202022.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/d4081516d55e1da0e45220ec475cc616d9957a7b/governance/votes/Executive%20vote%20-%20October%2012%2C%202022.md)"
```

Comment:
Not blocking but the `-q` flag here will squelch the download chatter on wget
```suggestion
    // Hash: cast keccak -- "$(wget -q https://raw.githubusercontent.com/makerdao/community/d4081516d55e1da0e45220ec475cc616d9957a7b/governance/votes/Executive%20vote%20-%20October%2012%2C%202022.md)"
```

Reply:
Naz is going to PR in a better version of this with the -q flag - not blocking but I will note this.

Reply:
Ah, `-q -O - 2>/dev/null)` is missing from the comment above.